app/helpers.js

```diff
@@ -1,4 +1,11 @@
-const { MAP_ORDER } = require('./constants');
+const bcrypt = require('bcryptjs'),
+  salt_sync = require('../config').encryption,
+  { MAP_ORDER } = require('./constants');
+
+exports.encryptPassword = async password => {
+  const salt = await bcrypt.genSalt(salt_sync);
```

Comment:
Is this working? According to documentation [async functions require a callback](https://www.npmjs.com/package/bcryptjs#gensaltrounds-seed_length-callback)

Reply:
In order to promisify a callback function you can wrap it into a Promise or use [Util.promisify](https://nodejs.org/dist/latest-v8.x/docs/api/util.html) package from node